assets/css/index.css

```diff
@@ -69,6 +69,33 @@ rh-footer {
   }
 }
 
+#home{
```

Comment:
i'm planning to make some changes. mostly code formatting but also some fixes so the HTML